DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -438,31 +438,13 @@ public void SetDelegateUserLearningHubAuthId_correctly_sets_delegates_learningHu
                 .And.Subject.Should().Be(learningHubAuthId);
         }
 
-        [Test]
-        public void GetDelegateUserLearningHubAuthId_returns_expected_delegate_learningHubAuthId()
-        {
-            using var transaction = new TransactionScope();
-
-            // Given
-            const int delegateId = 11;
-            const int expectedLearningHubAuthId = 5678;
-            userDataService.SetDelegateUserLearningHubAuthId(delegateId, expectedLearningHubAuthId);
-
-            // When
-            var result = userDataService.GetDelegateUserLearningHubAuthId(delegateId);
-
-            // Then
-            result.Should().NotBeNull()
-                .And.Subject.Should().Be(expectedLearningHubAuthId);
-        }
-
         [Test]
         public void GetDelegateUserLearningHubAuthId_returns_null_delegate_learningHubAuthId()
         {
             using var transaction = new TransactionScope();
 
             // Given
-            const int delegateId = 11;
+            const int delegateId = 3;

```

Comment:
Why the change?

Reply:
There was no reason to target different users for the two tests. Either should produce the same result.